view/sale_channel_form.xml

```diff
@@ -28,6 +28,14 @@
             <label name="price_list"/>
             <field name="price_list"/>
         </page>
+        <page string="Product Defaults" id="product_defaults">
```

Comment:
This page needs a big description in the top that the values are used when new products are created on imports from the external channel.